### ISSUE TITLE
Analyze GSV capture dates in our deployment cities

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2985]
I just saw a March 2022 pano in Oradell, NJ. I'm curious what the GSV capture dates look like there and if we've labeled on streets that have now received newer GSV scans. I'd like to re-run the same GSV capture data analysis we did in Oradell (see below). In the future, it would be interesting to schedule re-audits based on the availability of new GSV data.

![image](https://user-images.githubusercontent.com/1621749/182468784-f07f774e-f1fd-4a1c-9fc1-a3823f2e94eb.png)

This image was generated by a yet-to-be-public repo: https://github.com/jonfroehlich/gsv-capture-dates

**Comment by jonfroehlich** [comment]
This is also a potential research project with @crescendochu and Dr. Mooney.

**Comment by jonfroehlich** [comment]
We are now doing this with the Berkeley Data Science program

**Comment by misaugstad** [comment]
Agreed — this is covered now, and it went further than a one-off analysis.

Capture dates are no longer something we have to go analyze offline: #4384 built a standing imagery-freshness pipeline that stores per-street capture dates in `street_imagery`, fed both by panos labelers happen to view and by a nightly poll of each city's imagery provider (PRs #4649, #4650, #4651, merged 2026-08-15). So every deployment city continuously tracks its own imagery age.

The admin tooling you're thinking of is the **Imagery** page (`/admin/imagery`, #4908 / PR #4954): re-audit frontier, region ranking, next streets to poll, pipeline health, and a "how stale our labels are" chart.

And the "schedule re-audits based on new GSV data" wish from the description is live too — audits get flagged `outdated_imagery` when newer imagery lands, which downweights the street in the priority formula so labelers get routed back to it, surfaces as `km_needs_reaudit` in metrics and the v3 API, and shows up on the user dashboard as your streets needing a re-audit (#4896 / PR #4956).

(Per [this comment](https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2985#issuecomment-1769512321), the research-side analysis also happened with the Berkeley Data Science program.)
